Fundamentals/first_letter.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2016 Ami . All rights reserved.
+
+
+def user_input():
+    password = input("Tell me your password:")
+
+    print("The first letter you entered was: " + password[0].upper())
```

Comment:
Please extract a function that receives a password and prints the first letter. This way it would be easier to unit test later (conceptually).